File: sketches/sketches.model/src/main/java/edu/kit/kastel/lissa/sketches/model/ISketch.java
package edu.kit.kastel.lissa.sketches.model;

import edu.kit.kastel.lissa.sketches.model.elements.IBox;
import edu.kit.kastel.lissa.sketches.model.elements.IRelation;

public interface ISketch {
	IBox getCurrentInterpretation(IBox element);

	IRelation getCurrentInterpretation(IRelation relation);
}

File: sketches/model/src/main/kotlin/edu/kit/kastel/lissa/sketches/model/ISketch.kt
package edu.kit.kastel.lissa.sketches.model

import edu.kit.kastel.lissa.sketches.model.elements.IBox
import edu.kit.kastel.lissa.sketches.model.elements.IRelation

interface ISketch {
    fun getCurrentInterpretation(element: IBox): IBox?
    fun getCurrentInterpretation(relation: IRelation): IRelation?
}